content/ember-v4-typescript-support-update.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Ember v4 TypeScript Support Update
+authors:
+  - chriskrycho
+date: 2022-01-25T00:00:00.000Z
```

Comment:
This is a tentative/projected date. The PR is huge and will need to be reviewed and merged by the DefinitelyTyped team, and that *could* be as soon as today but it could be after this projected date, too!

Reply:
```suggestion
date: 2022-01-25T18:50:00.000Z
```